cookbook/templating/namespaced_paths.rst

```diff
@@ -74,6 +74,10 @@ directory:
             );
         ));
 
+.. caution::
+
+    These paths are not warmed during cache warming. This is fixed in 2.8.
```

Comment:
I don't find this explanation clear enough, specially for newcomers. What is "warmed" supposed to mean? What side-effects will this "bug" introduce in my applications?